src/mscorlib/System.Private.CoreLib.csproj

```diff
@@ -283,6 +285,23 @@
     <Compile Include="$(BclSourcesRoot)\System\Runtime\Intrinsics\X86\Sse42.cs" />
     <Compile Include="$(BclSourcesRoot)\System\Runtime\Intrinsics\X86\Ssse3.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(Platform)' != 'amd64' AND '$(Platform)' != 'x86'">
+    <Compile Include="$(BclSourcesRoot)\System\Runtime\Intrinsics\X86\AesFallback.cs" />
```

Comment:
Nit: Our convention is to have dot between the name and suffix for pattern like these. Also, fallback suffix maybe a bit misleading because the implementation is not actually true fallback - it just throws PNSE.

https://github.com/dotnet/corefx/blob/master/src/System.CodeDom/src/Microsoft/CSharp/CSharpCodeGenerator.PlatformNotSupported.cs is a good prior art. Call this `Aes.PlatformNotSupported.cs` ?

Reply:
It maybe a good idea to submit the `*.cs` changes in separate PR to make the rest of the actual changes in this PR easier to review.

Reply:
Thanks, this name makes more sense.

> It maybe a good idea to submit the *.cs changes in separate PR to make the rest of the actual changes in this PR easier to review.

Will do.